services/supervisor/src/api.rs

```diff
@@ -251,15 +266,12 @@ impl PrivateApi for ApiImpl<'_> {
     fn migration_status(
         &self,
         query: MigrationInfoQuery,
-    ) -> Result<ProcessStateResponse, Self::Error> {
+    ) -> Result<MigrationStateResponse, Self::Error> {
```

Comment:
Do you think so? Usually I poll the exonum API with `curl`, so when I see 404, I start looking for the mistake in URI. IMHO, it will cause more confusion (at least for me personally).

Reply:
AFAIK, several APIs in other services work like that (e.g., the wallet endpoint in the cryptocurrency demos). One can argue that requesting a non-existing migration *is* wrong, since it should not happen with properly designed web front-end.

Reply:
Indeed. Well, we now have a pretty good API to build an error description, so it's a better alternative.